cax/tasks/process.py

```diff
@@ -25,7 +25,7 @@ def get_pax_hash(pax_version, host):
     if pax_version == 'head':
         git_args = "--git-dir=" + PAX_DEPLOY_DIRS[host] + "/.git rev-parse HEAD"
     else:
-        git_args = "--git-dir=" + PAX_DEPLOY_DIRS[host] + "/.git rev-parse " + pax_version
+        git_args = "--git-dir=" + PAX_DEPLOY_DIRS[host] + "/.git rev-parse HEAD"
```

Comment:
You don't want to change this.  This is something @pdeperio added for getting the hash of versions (not head)